tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
@mstajdohar The last suggestion does't work.

I found the committed solution as a workaround for the fact that the gzip.open() does not support universal new line mode.